lib/howl/io/file.moon

```diff
@@ -201,6 +202,17 @@ class File extends PropertyObject
 
     files, false
 
+  copy: (dest, flags, progress) =>
+    bitflags = 0
+    flags or= {}
+
+    for flag in *flags
```

Comment:
There's actually a helper down in ljglibs that can be used for doing this instead, I'll give some more details on this for ljglibs `File` below. We don't have any use of `progress` right now, so I say we should leave that out for now.